README.md

```diff
@@ -1,6 +1,6 @@
 Updated: November 25, 2020. The spec in this repository is now the most up-to-date version of GTFS-Flex. The code in this repo contains the official proposal for GTFS-Flex v2, a GTFS extension that covers all demand-responsive services for the purposes of discovery in trip planning. All features of this specification proposal are currently produced by [Trillium](https://trilliumtransit.com/) and consumed by [OpenTripPlanner version 2](https://www.opentripplanner.org/). DemandTrans and IBI group expect to produce data in this spec by early 2021.
 
-"Version 1" of the GTFS-Flex specification is utilized by OTP 1.4. You can review the Version 1 specification by reviewing versions of this repository from before October 2020.
+"Version 1" of the GTFS-Flex specification is utilized by OTP 1.4. You can review the Version 1 specification by reviewing versions of this repository from before October 2020. [See the earlier version here.](https://github.com/scmcca/gtfs-flex/blob/master/spec/reference.md)
```

Comment:
Is it strange that the link here is to my fork of this repo? It seems that my fork has not been updated since the merging of v2 in the main, which is functionally correct if referring to v1. But if I update my fork or delete it, this link will be irrelevant. Please see my counter suggestion at #45 to make use of GitHub's versioning tags :)